src/app/public/modules/list-column-selector-action/list-column-selector-button.component.html

```diff
@@ -0,0 +1,11 @@
+<button
+  class="sky-btn sky-btn-default"
+  type="button"
+  [attr.aria-label]="'skyux_grid_columns_toolbar_button' | skyLibResources"
+  [title]="'skyux_grid_columns_button' | skyLibResources"
+>
+  <sky-icon icon="columns"></sky-icon>
+  <span class="sky-column-selector-action-btn-text">
+    {{ 'skyux_grid_columns_toolbar_button' | skyLibResources }}
+  </span>
```

Comment:
☝️ There's a few attributes in here that need to be on new lines.

Reply:
Done